smart_open/smart_open_lib.py

```diff
@@ -615,3 +615,8 @@ def _encoding_wrapper(fileobj, mode, encoding=None, errors=DEFAULT_ERRORS):
     else:
         decoder = codecs.getwriter(encoding)
     return decoder(fileobj, errors=errors)
+
+def _add_sheme_to_host(host):
+    if (host.startswith('http')):
```

Comment:
unnecessary outer brackets, please remove it

Reply:
Also, if the host is `httpsomething.com`, what will happen? Probably you need to check "http://" instead.